doc/release/1.3.0-notes.rst

```diff
@@ -6,9 +6,9 @@ SciPy 1.3.0 Release Notes
 
 .. contents::
 
-SciPy 1.3.0 is the culmination of X months of hard work. It contains
+SciPy 1.3.0 is the culmination of 5 months of hard work. It contains
 many new features, numerous bug-fixes, improved test coverage and better
-documentation. There have been a number of deprecations and API changes
+documentation. There have been some API changes
```

Comment:
no deprecations listed for this release! I almost think it is not worth removing those words because of our (or at least my) tendency to copy / template release notes upward across versions

Reply:
we have that, it's committed to master again right after branching (search for `REL:` commits). If there are no deprecations then this text shouldn't say that, so agree with change in this PR